src/core/core/library_utils.cpp

```diff
@@ -71,7 +71,7 @@ namespace dsn {
             }            
 # elif defined(__linux__) || defined(__FreeBSD__) || defined(__APPLE__)
             module_name = "lib" + module_name + ".so";
-            auto hmod = dlopen(module_name.c_str(), RTLD_LAZY);
+            auto hmod = dlopen(module_name.c_str(), RTLD_LAZY|RTLD_GLOBAL);
```

Comment:
@qinzuoyan let's stick to the GLOBAL solution for the time being. You concern is definitely correct, and we can fix this later when we have a better solution. Right now, the solution we see is to link python with our main program, which is not acceptable.